.github/pull_request_template.md

```diff
@@ -30,6 +30,7 @@ You may intentionally leave this section blank and remove the title.
 - [ ] User-facing API changes need the "User-facing API Change" label.
 - [ ] Release notes should be added as a separate file under `docs/release-notes/`.
 See [Release Note](../docs/release-notes/README.md) for details.
+- [ ] Licenses should be included for new code which copied and/or modified from any external code.
```

Comment:
nit: IMO either of these looks better:

for **any** new code ~which~ copied and/or modified from 

or which **was** copied

Reply:
done